lib/logging.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+#
+# Logging Library
+# Forked from
+# https://github.com/juan131/bash-libraries/blob/master/lib/liblog.bash
+
+# Color palette
+RESET='\033[0m'
+GREEN='\033[38;5;2m'
+RED='\033[38;5;1m'
+YELLOW='\033[38;5;3m'
+
+source config.sh
```

Comment:
📝 **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not following: config.sh: openBinaryFile: does not exist (No such file or directory) [SC1091](https://github.com/koalaman/shellcheck/wiki/SC1091)
<!-- __reviewdog__:ChA5OWI1Y2ZhMTk2NTJlZTlmEgpzaGVsbGNoZWNr -->